AUTHORS

```diff
@@ -1,3 +1,3 @@
 Nicholas H.Tollervey (ntoll@ntoll.org)
 Matt Wheeler (m@funkyhat.org)
-
+Edmond Locard (L0C4RD@gmail.com)
```

Comment:
Aha - I did _not_ run `make check`! Apologies, I'll do that on my end and try to get those warnings cleaned up for you, along with those compatibility problems.

The `IntelHex` module was just a shortcut for quickly interpreting Intel Hex files, but everything it accomplishes can be achieved with a dict and some parsing code. I'll implement this and pop it into the next commit, so it shouldn't jeopardise uflash's standalone-ness. 😄